cmd/edit/cluster/cmd.go

```diff
@@ -233,7 +234,7 @@ func run(cmd *cobra.Command, _ []string) {
 		private = &privateValue
 	} else if privateValue {
 		r.Reporter.Warnf("You are choosing to make your cluster API private. %s", privateWarning)
-		if !confirm.Confirm("set cluster '%s' as private", clusterKey) {
+		if !confirm.Yes() && !confirm.Confirm("set cluster '%s' as private", clusterKey) {
```

Comment:
I think we only needed to add the flag due to this

https://github.com/openshift/rosa/blob/510a8c7e2c25b85a2ec5e30be3c0deeaa5d44ac9/pkg/interactive/confirm/confirm.go#L50

Would you have some time to test removing the confirm.yes?

Reply:
After more exploration, I think you are right. Because the "confirm.confirm" returns yes if the flag has been passed, we do not seem to need the "confirm.yes" check. I have tested after removing this check and the confirmation question is still not being asked